### DISCUSSION TITLE
Can I use pyleniumio to start browser with user data

### DISCUSSION THREAD

**Discussion by nguyet0310** [ElSnoMan/pyleniumio#262] [General]
I need to start a browser with user profile data, is it possible to configure in pylenium.json?

**Comment by ElSnoMan** [comment]
@nguyet0310 How would you normally do this with Selenium?

**Reply by nguyet0310** [reply]
I follow this guide: https://stackoverflow.com/questions/50635087/how-to-open-a-chrome-profile-through-user-data-dir-argument-of-selenium
But with pelynium.json, I can't find any where to add arguments for chrome options: 
<img width="291" alt="ảnh" src="https://user-images.githubusercontent.com/32532761/170847272-b48e2909-9688-4357-a10b-08ad83ded84b.png">

**Reply by ElSnoMan** [reply]
Driver "arguments" are part of the Options object, so you use the `"options"` field in `pylenium.json`. For example:

```py
"options": ["--disable-extensions", "other-option",]
```

**Reply by ElSnoMan** [reply]
More info can be found in the official docs
https://docs.pylenium.io/configuration/driver

**Reply by nguyet0310** [reply]
yeah, it works, thank you :)
